charts/opentelemetry-operator/values.schema.json

```diff
@@ -621,6 +621,14 @@
                             "examples": [
                                 "true"
                             ]
```

Comment:
Another more flexible option is to allow any with string type
``` 
"additionalProperties": {
                      "type": "string"
                    },
```

that would block me to add wrong values:
```
- manager.env.CUSTOM_ENV: Invalid type. Expected: string, given: integer

- manager.env.CUSTOM_ENV: Invalid type. Expected: string, given: object
 
```

Reply:
Ya this section shouldnt be restricted at all, you should be able to put whatever env vars you want. The schema shouldn't have any required values and should allow additional properties

Reply:
changed 👍 

I still kept the type since the `deployment.yaml` expects it to be a key/value